news/2020-11-26-community-mentorship-program-winter-2021-signups-now-open.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Community Mentorship Program Winter 2021 Signups Now Open
+date: 2020-11-26 08:00:00 +0000
+---
+
+With holiday cheer and a upcoming new year, the 17th cycle of the Community Mentorship program is on its way! If you've ever found yourself wanting to expand your current knowledge and skills or help beginner mappers get started, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills under the guidance of more experienced mappers and modders in the community. 
+
+Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events (like mapping contests), hang out with fellow mentees, and more!
+
+If you're experienced in mapping and have a desire to take those who are new under your wing, then give joining us as Mentor a shot! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1188335), you'll be able to apply right away. If you're only just getting into mapping or are eager to hone your skills, then signing up as Mentee might just be your way to do it!
+
+---
+
+For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
+
+- Mentor signups will open on November 30th and close on December 6th.
+- Mentee signups will open on December 27th and close on January 3rd.
+- The 2021 Winter cycle will run from January 4th to April 9th.
```

Comment:
```suggestion
- Mentor signups will open on **November 30th** and close on **December 6th**.
- Mentee signups will open on **December 27th** and close on **January 3rd**.
- The 2021 Winter cycle will run from **January 4th** to **April 9th**.
```
just thought these start/end dates could use some bold formatting, feel free to dismiss this one though - just a suggestion